src/amo/utils.js

```diff
@@ -1,8 +1,31 @@
-import { LANDING_PAGE_ADDON_COUNT } from 'amo/constants';
+import {
+  FEATURED_ADDONS_TO_LOAD,
+  LANDING_PAGE_ADDON_COUNT,
+} from 'amo/constants';
+import { getFeatured, loadFeatured, failFeatured } from 'amo/actions/featured';
 import { getLanding, loadLanding, failLanding } from 'amo/actions/landing';
 import { featured as featuredAPI, search } from 'core/api';
 
 
+export function fetchFeaturedAddons({ addonType, api, dispatch }) {
+  dispatch(getFeatured({ addonType }));
+
+  const filters = { addonType, page_size: FEATURED_ADDONS_TO_LOAD };
+
+  return featuredAPI({ api, filters })
+    .then((response) => dispatch(
+      loadFeatured({ addonType, ...response })
```

Comment:
I'd rather see `entities` and `result` instead of `...response`. Since there are only two keys I don't think it saves much to use a spread.